src/rgw/rgw_rest.cc

```diff
@@ -1139,18 +1138,17 @@ int RGWPutObj_ObjStore::get_data(bufferlist& bl)
 
   int len = 0;
   if (cl) {
-    ACCOUNTING_IO(s)->set_account(true);
+    /* There is no need to manage the accounting enablement here. It's now
```

Comment:
As per coding guidelines Please use "It is" instead of it's.
A PR is already raised and approved for it.